### PR TITLE
Sidebar design with White Side Nav

### PR DESCRIPTION
**Pull request by rtluu**

### CODE REVIEW

File: src/modules/queue/components/Queue.jsx [hunk 1]
```diff
@@ -36,14 +38,7 @@ const Sidebar = styled.div`
 	height: 100%;
 	transition: all 0.1s;
 	right: 0;
-	width: 0;
-	z-index: 100;
```
[the remark this answers is not in the document]
Comment (rtluu) [reply src/modules/queue/components/Queue.jsx]: My bad, I did that during testing and forgot :/. I'll be more careful

File: src/modules/uniplayer/uniplayer.scss [hunk 2]
```diff
@@ -1,197 +1,195 @@
-.uniplayer {
-	background: #8e1d2e;
-	bottom: 0px;
-	left: 0px;
-	position: fixed;
-	right: 0px;
-	top: 0px;
+.uniplayer-outer {
```
[the remark this answers is not in the document]
Comment (rtluu) [reply src/modules/uniplayer/uniplayer.scss]: Yes for sure, I'm happy to walk you through it and work to refactor it to make it simpler

File: src/modules/uniplayer/Search/YoutubeResult.js [hunk 3]
```diff
@@ -3,18 +3,19 @@ import React from 'react';
...
 export default function YoutubeResult({
 	onClick,
+	highRes,
 	result: {
 		title,
-		description,
 		thumbnails: {
-			default: {url, height, width}
+			default: {url: defaultUrl},
+			high: {url: highUrl}
```
[the remark this answers is not in the document]
Comment (rtluu) [reply src/modules/uniplayer/Search/YoutubeResult.js]: Just tested and it's great, thanks for your help!

File: src/modules/queue/components/Queue.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jamesmcnamara) [reply src/modules/queue/components/Queue.jsx]: <3

**Pull request merged**